Anlab.Mvc/Controllers/AdminController.cs

```diff
@@ -170,7 +170,13 @@ public IActionResult ListUsersOrders(string id)
             {
                 var orderToUpdate = await _dbContext.Orders.SingleAsync(a => a.Id == model.OrderId.Value);
 
-                await _orderService.PopulateOrder(model, orderToUpdate);
+                await _orderService.PopulateOrder(model, orderToUpdate, true);
```

Comment:
@srkirkland That is just to tell me where the service is called from. I don't want any of those fields updated from the client side. But I'll think about it.